requirements.md

```diff
@@ -91,3 +91,37 @@
   User pastes unformatted text into the search bar
   1. User pastes unformatted text into the search bar
   2. System automatically formats text to default text and font
+
+5. Use Case Name: User Profile
+- **Pre-condition:**  User has access to wifi and has an account already created.
+ 
+- **Trigger:**  User clicks profile icon
+ 
+- **Primary Sequence:**
+  1. User wants to view their profile
+  2. User clicks on user icon
+  3. System identifies their account
+  4. System navigates user to their personal user profile
+ 
+- **Primary Postconditions:** User is able to view their personal profile and interact with their page.
+ 
+- **Alternate Sequence:**
+  1. User has not created a profile yet when clicking profile icon
+  2. System prompts user with create a profile screen
```

Comment:
I'm not too sure I follow. For an alternate sequence for "User Profile" you want to include selecting files to make a post, then redirect them to the home page?

Reply:
I meant that the user clicks on a wrong button (make post button) instead of clicking on their profile icon. That's they they eventually close it and go back to home page.

I think your alternative sequence is actually better but I'm not too sure about it because profiles are usually created right after you create an account. That's why I suggested another sequence instead.